src/js/angular/ttyg/templates/chat-panel.html

```diff
@@ -35,12 +38,15 @@
                         <i class="fa-regular fa-wand-magic-sparkles"></i>
                     </div>
                 </div>
+                <div class="assistant-progress" ng-if="!chatItem.answer">
+                    <div onto-loader-new size="40"></div>
+                </div>
             </div>
         </div>
     </div>
     <div class="mew-question mb-3 mt-2 d-flex">
-        <input class="form-control question-input" ng-model="chatItem.question.message" type="text">
-        <button class="btn btn-primary ask-button" ng-disabled="!chatItem.question.message || !selectedAgent || selectedAgent.isDeleted" ng-click="ask()">
+        <input class="form-control question-input" ng-model="chatItem.question.message" type="text" ng-disabled="asking || loadChat">
```

Comment:
loadChat should be `loadingChat` or something

Reply:
done